docs/content/guides/7.multistore/3.patterns/5.subpath/1.subpath.md

```diff
@@ -0,0 +1,696 @@
+::
+
+Make sure that all your internal links and navigation components preserve the store context by including the current configId in the URLs. This applies to:
+
+- Navigation menus
+- Product links
+- Category links
+- Cart and checkout flows
+- Footer links
+- Breadcrumbs
+- Any other internal navigation elements
+
+Without this, users might unexpectedly switch between stores while browsing your site.
+
+## Applying Store-Specific Styling
+
+Now that you have CSS variables defined for each store, you need to apply the store identifier as a CSS class to enable the different styles.
+
+::tabs{:titles='["Next.js", "Nuxt"]'}
+
+#tab-1
+In your Next.js layout, apply the configId as a class:
+
+```tsx
+// apps/storefront-unified-nextjs/app/[locale]/[configId]/layout.tsx
+import { PropsWithChildren } from 'react';
+
+interface LayoutProps extends PropsWithChildren {
+  params: {
+    configId: string;
+  };
+}
+
+export default function Layout({ children, params: { configId } }: LayoutProps) {
+  return (
+    <div className={configId}>
+      {children}
+    </div>
+  );
+}
+```
+
+#tab-2
+In your Nuxt app.vue, apply the configId as a class:
+
+```vue
+// apps/storefront-unified-nuxt/app.vue
+<template>
+  <Body :class="['font-body', route.params.configId]" />
+  <NuxtLayout>
+    <NuxtPage />
+  </NuxtLayout>
+</template>
+
+<script setup lang="ts">
+const route = useRoute();
+</script>
+```
+::
+
+## Styling Components for Different Stores
+
+With the store identifier applied as a class, you can now customize components with store-specific styles. You'll need to update your Tailwind configuration to support this.
+
+::tabs{:titles='["Next.js", "Nuxt"]'}
+
+#tab-1
+First, update your Tailwind config:
+
+```ts
+// packages-end-user/tailwind-config/src/nextjs.ts
+import { tailwindConfig } from "@storefront-ui/react/tailwind-config";
+import type { Config } from "tailwindcss";
+import plugin from "tailwindcss/plugin";
+
+const config: Config = {
+  // ...
+  plugins: [
+    // ...
+    plugin(({ addVariant }) => {
+      // Add variant for electronics store
+      addVariant("electronics", ".electronics &");
+      // Add variant for apparel store
+      addVariant("apparel", ".apparel &");
+    }),
+  ],
+  // ...
+};
+export default config;
+```
+
+Then use these variants (`electronics:`, `apparel:` etc.) in your components:
+
+```tsx
+// apps/storefront-unified-nextjs/components/navigations/navbar-top.tsx
+export default function NavbarTop({ children, className, filled }: NavbarTopProps) {
+  return (
+    <header
+      className={classNames(
+        'sticky top-0 z-40 flex h-14 bg-primary-800 electronics:!bg-primary-600 apparel:!bg-primary-400 md:-top-5 md:h-20 md:pt-2.5',
```

Comment:
I didn't know of this tailwind feature, will this example work? You have 3 backgrounds defined here? Do I understand correctly that it will try 800 then overwrite this with one of the variants if they are available? But then Idk how does it know which variant is the active one. Could you make it a bit more clear in the guide?

Reply:
it works, I've added short explanation

![CleanShot 2025-04-07 at 17 02 23@2x](https://github.com/user-attachments/assets/f9a20d08-71c5-4ed9-9533-7abbee98fdfe)